packages/textile/src/users.spec.ts

```diff
@@ -72,10 +72,12 @@ describe('Users...', () => {
         expect(err.code).to.equal(grpc.Code.NotFound)
       }
       // All good
+      ctx.withThreadName('foo')
```

Comment:
once https://github.com/textileio/js-threads/pull/263 is released, we can drop this

Reply:
hmm... that actually doesn't seem to be the case. i've done the update. everything else is now passing. if i remove this line it fails

Reply:
oop, looks like maybe i have a partial dep update. let me try that again.

Reply:
ooookay, pilot error. back on track